x-pack/plugins/security/public/management/edit_role/edit_role_page.tsx

```diff
@@ -4,6 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import _, { get } from 'lodash';
```

Comment:
__note:__ that view would definitely need to be refactored in the future. It's quite big and has lots of dependencies that blocks rendering. Technically we can render the view partially while loading some information in parallel, but I tried to keep the same behavior we had in the legacy plugin to reduce the risk of breaking something accidentally.

Reply:
Agree 100%, I'll take this on as its own task in a followup (unless you get bored!)